rust/operator-binary/src/main.rs

```diff
@@ -209,3 +224,20 @@ fn references_authentication_class(
         .iter()
         .any(|c| c.common.authentication_class_name() == &authentication_class_name)
 }
+
+fn references_config_map(
+    airflow: &DeserializeGuard<v1alpha1::AirflowCluster>,
+    config_map: &DeserializeGuard<ConfigMap>,
+) -> bool {
+    let Ok(airflow) = &airflow.0 else {
+        return false;
+    };
+    // Check for ConfigMaps that are referenced by the spec and not directly attached to a Pod
+    match airflow.spec.cluster_config.authorization.clone() {
+        Some(airflow_authorization) => match airflow_authorization.opa {
+            Some(opa_config) => opa_config.opa.config_map_name == config_map.name_any(),
+            None => false,
+        },
+        None => false,
+    }
+}
```

Comment:
We can drop the clone() (thanks @Techassi):

```suggestion
    match &airflow.spec.cluster_config.authorization {
        Some(airflow_authorization) => match &airflow_authorization.opa {
            Some(opa_config) => opa_config.opa.config_map_name == config_map.name_any(),
            None => false,
        },
        None => false,
    }
}
```